cep-0021.md

```diff
@@ -25,24 +25,25 @@ However, not all channels on the default server (conda.anaconda.org) provide thi
 
 Having two separate files also poses some problems as extra mechanisms have to be introduced in the build infrastructure to manage and sync both files on the build machines.
 
-## Specification
+## Discussion
 
 CEP 12 mentions the following reasons for splitting the information into two files:
 
 > * It would require extending the repodata schema, currently not formally standardized.
 > * It would increase the size of the already heavy repodata.json files.
 > * (Typed) repodata parsers would need to be updated to handle the new field.
+
 We propose that these reasons no longer hold with [sharded repodata](https://github.com/conda/ceps/blob/main/cep-0016.md).
 
-**It would require extending the repodata schema, currently not formally standardized.**
+### Re: "It would require extending the repodata schema, currently not formally standardized."
```

Comment:
The screenshot above also shows that the current highlight makes it seem like part of the regular text of the CEP. I don't care about the exact formatting, as long as it is clear that it's a point being addressed.